### ISSUE TITLE
Path unusable for copy

### ISSUE THREAD

**Issue by Akaizoku** [tweh/menukeys#51]
While the use of `menukeys` is great visually, it makes it impossible to re-use a path by selecting and copying it.

It would be great if it was possible to change the actual text behind the visual separators to re-use the same separator that was used to break the path appart.

**Comment by tweh** [comment]
This is a pretty nice idea, however at the moment unfortunately I don’t have the time to work on the packe, but I’ll keep it in mind!

**Comment by Skillmon** [comment]
This could be possible with the `accsupp` package. The result would depend on the used PDF viewer (I tested the `accsupp` package with a few a while ago and while some pick up on the `ActualText` others ignore it).

Maybe we could provide a package option to switch on/off the use of `accsupp`. The output routine would have to be changed for this to use something like

```latex
\BeginAccSupp{ActualText=<string>}%
  <original output routine>%
\EndAccSupp{}
```

We'd have to make sure that there is no page break between `\BeginAccSupp` and `\EndAccSupp`. Maybe we could even only apply it to the separator, which should be more stable (but a bit more work).

**Comment by tweh** [comment]
If we apply it only to the separator I fear it could happen that some viewers add extra spaces (from bevor after the separator) which would circumvent the whole effort of making the path copy-able.

**Comment by Skillmon** [comment]
Yes, this could happen indeed. I'll have to try things, maybe I can come up with something that works well across a bunch of viewers.
